src/UbiBuilder.h

```diff
@@ -25,10 +25,15 @@ Developed and maintained by Jose Garcia for IoT Services Inc
 #define _UbiBuilder_H_
 
 #include <map>
+
```

Comment:
There are many includes here that are not supposed to be. 
Check which of them are necessary.